src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeffUnitTest.java

```diff
@@ -452,4 +452,24 @@ public void testTinyCohorts_AS() {
     }
 
 
+    @Test
+    public void testRoundingIsntAppliedToEachGenotype() {
+        final VariantContext vc = new VariantContextBuilder("in memory", "1", 100, 100,
+                                                                 Arrays.asList(Aref, T)).make();
+        final ArrayList<Genotype> genotypes = new ArrayList<>();
+        final int numberOfSamples = 200;
+        for(int i = 0; i< numberOfSamples; i++){
+            final Genotype g = new GenotypeBuilder("sample" + i, Arrays.asList(Aref, T)).PL(new int[]{40,10,0}).make();
+            Assert.assertTrue(GenotypeUtils.isDiploidWithLikelihoods(g));
+            genotypes.add(g);
+        }
+
+        final Pair<Integer, Double> sampleCountAndCoefficientPair = InbreedingCoeff.calculateIC(vc,
+                                                                                    GenotypesContext.create(genotypes));
+        final int sampleCount = sampleCountAndCoefficientPair.getLeft();
+        final double inbreedingCoefficient = sampleCountAndCoefficientPair.getRight();
+        Assert.assertEquals(sampleCount, numberOfSamples);
+        final double expectedValueFromGATK3_7 = 0.0456;
+        Assert.assertEquals(inbreedingCoefficient, -0.0456, DELTA_PRECISION);
+    }
```

Comment:
This test fails if you temporarily turn rounding **on** in `InbreedingCoefficient`, correct?